scss/components/masonry-block/_masonry-block.scss

```diff
@@ -144,6 +144,44 @@
   &.stamp-3 {
     background: color(action);
   }
+
+  a {
+    text-align: center;
+    width: 100%;
+    color: color(white);
+
+    &:hover {
+      color: color(emphasis);
+    }
+  }
+
+  .button {
+    margin-bottom: 10px;
+    font-size: 1.6em;
+    border-color: color(white);
+    width: 100%;
+    text-align: center;
+
+    &:hover {
+      background-color: color(emphasis);
+      color: color(white);
+    }
+  }
+
+  .about {
+    width: 100%;
+    text-align: center;
+    font-size: .8em;
+  }
+
+  h3.masonry-block__title {
```

Comment:
Are we sure we want to hide them via css? Is it not possible to hide the title by entering only the correct fields?

Reply:
@buttonwillowsix 
 ^^^

Reply:
I wondered about that. But I think there are cases where they might want to unhide them (originally the "Subscribe" was the title on the comp. Anyhow, you need a title to find the darned thing, yes?

Reply:
I also considered leaving the title un-hidden, and just leaving that particular one hidden in CSS injector

Reply:
Yeah, at some point you had it removed in the template and then I think I had cold feet about taking it out.... I will remove that and take a look at the template.